src/security/PeerOptions.cc

```diff
@@ -738,6 +741,7 @@ Security::PeerOptions::updateContextCrl(Security::ContextPointer &ctx)
 #endif
 
 #endif /* USE_OPENSSL */
+    (void)ctx;
```

Comment:
This would disable a useful warning if a buggy `USE_OPENSSL` code modification drops `ctx`. I think we should add an `#else` use case here.

Reply:
ok

Reply:
This change request was marked as resolved, but the problem is still there AFAICT.

Reply:
Fixed in commit f2de1d91c

Reply:
> Fixed in commit f2de1d91c

That commit places the use-ctx code inside `#if` that already uses ctx. The use-ctx code should be placed inside a currently missing `#else` clause instead, as mentioned in the original change [request](https://github.com/squid-cache/squid/pull/667/files#r632119084).